Admin/AbstractAdmin.php

```diff
@@ -3112,7 +3112,11 @@ public function canAccessObject($action, $object)
         );
 
         if ($this->adminBuilder === null) {
-            $this->adminBuilder = new AdminBuilder();
+            $this->adminBuilder = new AdminBuilder(
+                $this->getShowBuilder(),
+                $this->getListBuilder(),
+                $this->getDatagridBuilder()
```

Comment:
I could only find interfaces for those 3 classes. Is there any actual implementation of a ShowBuilder o ListBuilder?